index.html

```diff
@@ -137,7 +141,56 @@ <h2>Currency Converter</h2>
         </fieldset>
       </form>
     </div>
-    <div></div>
+
+    <!-- **********************************************************-->
+    <!-- Week2 / An aaray of the currency rate objects -------------->
+    <!-- **********************************************************-->
+    <br />
+    <div id="array-of-currencies-section-beta">
+      <form id="brand-currency-form-beta">
+        <fieldset>
+          <legend>Currency rate creator beta</legend>
+
+          <!-- base currency selection -->
+          <label for="base-currency-beta">Choose a base currency:</label>
+          <select id="base-currency-beta">
+            <option value="EUR">"EUR"</option>
```

Comment:
`<option value="EUR">"EUR"</option>`

there is no need for quotes between tags `>"EUR"<` please remove

Reply:
Sorry that was a mistake. 
Done. Thank you.